### PR TITLE
Update office-365-microsoft-defender-antivirus.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the broken link

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6903

File: microsoft-365/security/defender-endpoint/office-365-microsoft-defender-antivirus.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ Microsoft Defender for Office 365 integrated with Microsoft Defender for Endpoin
 
 - Once integration is enabled, your security operations team can see a list of devices that are used by the recipients of any detected URLs or email messages, along with recent alerts for those devices, in the <a href="https://go.microsoft.com/fwlink/p/?linkid=2077139" target="_blank">Microsoft 365 Defender portal</a>.
 
-If you haven't already done so, [integrate Microsoft Defender for Office 365 with Microsoft Defender for Endpoint](/microsoft-365/security/office-365-security/integrate-office-365-ti-with-wdatp).
+If you haven't already done so, [integrate Microsoft Defender for Office 365 with Microsoft Defender for Endpoint](/microsoft-365/security/office-365-security/integrate-office-365-ti-with-mde)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/office-365-microsoft-defender-antivirus.md]:
```suggestion
If you haven't already done so, [integrate Microsoft Defender for Office 365 with Microsoft Defender for Endpoint](/microsoft-365/security/office-365-security/integrate-office-365-ti-with-mde).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Pull request merged**